dvc/command/ls/__init__.py

```diff
@@ -14,15 +17,21 @@ def run(self):
         from dvc.repo import Repo
 
         try:
-            nodes = Repo.ls(
+            fs_paths = Repo.ls(
```

Comment:
`fs_paths` is a confusing name, even for me. Maybe there is a better one?

Reply:
Agreed! :+1:  
I like `files`, but it doesn't describe the output well, IMHO

### TLDR
That's the reason why I didn't want to return object/dictionary from the method. The dict is specific
```
{
    "path": str,
    "isexec": bool,
    "isout": bool,
    "isdir": bool,
}
```
and uses only in one place. Hence we introduce additional structure that should have name, but it all names don't describe accurate the object.
I came from the main question - what this dict describes? I see the path where some `node`  (where `node` is file, dir or out) exists/can exist related repo with type of the node
Cause I am already using [fs_node](https://github.com/iterative/dvc/blob/bf4fe8aa3996cf4e8b65a0c1ac7bdfac7b0a3d14/dvc/repo/ls.py#L135) in `Repo.ls` with `path_info` (what means we really have such file on the path except out) during conversion to simple `str` - I needed to differentiate them somehow: https://github.com/iterative/dvc/blob/bf4fe8aa3996cf4e8b65a0c1ac7bdfac7b0a3d14/dvc/repo/ls.py#L64
So cause I use such name in `Repo.ls` I decided to keep it here